### ISSUE TITLE
[Windows] Commands create popup cmd windows on each call

### ISSUE THREAD

**Issue by clytras** [nathanbabcock/ffmpeg-sidecar#40]
On Windows 10, I get several command popup windows when running commands such as `ffmpeg_version`, `check_latest_version`, and `download_ffmpeg_package`.
![cmd popup window](https://github.com/user-attachments/assets/1c1a797c-5747-489f-8580-a0e9bb1afca4)

I see that for the `FfmpegCommand` there is a `create_no_window` method that I think it does that, it flags the command process to create no windows or keep them hidden.

Could we do the same for all other commands like `curl`, `curl_to_file` etc.?

**Comment by nathanbabcock** [comment]
Yes, absolutely. The fix that `create_no_window` applies is this:

```
std::os::windows::process::CommandExt::creation_flags(command, 0x08000000);
```

We can apply that to all other `Command`'s used in the library, and probably make it the default behavior as well.

**Comment by bliongosari1** [comment]
Hi @nathanbabcock is there a workaround to this at the moment or calling this already fixes it for everything?

`std::os::windows::process::CommandExt::creation_flags(command, 0x08000000);`

Thanks for the help

**Comment by nathanbabcock** [comment]
@clytras @bliongosari1 I've prepared a PR, you could begin using it right away with this temporary change in your `Cargo.toml`:

```toml
ffmpeg-sidecar = { git = "https://github.com/nathanbabcock/ffmpeg-sidecar.git", branch = "fix/windows-creation-flags" }
```

If you find any lingering issues with it please let me know, otherwise I'll intend to merge & publish it this weekend.

**Comment by clytras** [comment]
Hey @nathanbabcock, that's awesome.

My use case is with Tauri. When running in dev mode, there is the development shell that everything spawns there, but when running the production release there is no shell so it opens cmd windows.

I will try it now and I'll report back.

BTW, this wrapper works pretty good, very nice job!

**Comment by clytras** [comment]
I can confirm that everything, except `ffprobe_version`, works without poping any windows.
I think it's because of `ffprobe_version_with_path` command missing `create_no_window` on the PR.

https://github.com/nathanbabcock/ffmpeg-sidecar/blob/1f2692dd1755a3c6991fe9f7d40984556adb161a/src/ffprobe.rs#L45-L47

**Comment by nathanbabcock** [comment]
Great catch, thank you!
